src/main/java/uk/gov/verifiablelog/merkletree/MerkleUtil.java

```diff
@@ -6,15 +6,17 @@
 /**
  * A utility class that provides commonly used operations when working with Merkle Trees.
```

Comment:
I see this class as *not* part of the public interface of the library, so we can maybe document that anything may change at any time